resources/js/app.js

```diff
@@ -1550,6 +1552,16 @@ jQuery(document).ready(function () {
     lastScrollPosition = currentScrollPosition
   })
 
+  // Sentry error
+  Sentry.init({
+    Vue,
+    dsn: "https://50fd2fa440af4bb4a230f40ca8d8cf90@o879179.ingest.sentry.io/5831645",
```

Comment:
Will this report errors in local dev too?

Perhaps that's OK?

Reply:
See above.